qbft/spectest/tests/roundchange/justification_wrong_value.go

```diff
@@ -1,34 +0,0 @@
-package roundchange
-
-import (
```

Comment:
why deleted test?

Reply:
I thought it needs to be deleted because we don't have the justification data field. Should I change the test logic instead of deleting it so that the test checks for invalid full data?

Reply:
we do, please reinstate the test

Reply:
Done